Content/src/Server/ServerGiraffe.fs

```diff
@@ -1,35 +1,44 @@
-open System
-open System.IO
-open System.Threading.Tasks
-
+#if (remoting)
+open Fable.Remoting.Server
```

Comment:
I prefer sorting not alphabetically, but rather by modules first, e.g.:

1. .net 
1. asp
1. 3rd party libs
1. own modules

Reply:
Mmmmm. This is subjective, so I'll undo the changes I made here - we can discuss this in another PR  :-)